src/main/java/hyperledger/besu/java/rest/client/dto/TopicFilter.java

```diff
@@ -0,0 +1,38 @@
+package hyperledger.besu.java.rest.client.dto;
+
+import hyperledger.besu.java.rest.client.config.EthEventsProperties;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.web3j.protocol.core.DefaultBlockParameter;
+import org.web3j.protocol.core.methods.request.EthFilter;
+
+@Slf4j
+@Data
+@Builder
+public class TopicFilter {
+  private DefaultBlockParameter fromBlock; // optional, params - defaults to latest for both
+  private DefaultBlockParameter toBlock;
+  private String blockHash; // optional, cannot be used together with fromBlock/toBlock
```

Comment:
Do we need these parameters in our case ?  Are these getting set anywhere ?

Reply:
addressed